tools/downloader/downloader.py

```diff
@@ -234,6 +242,15 @@ def try_retrieve(reporter, destination, model_file, cache, num_attempts, start_d
     return success
 
 def download_model(reporter, args, cache, session_factory, requested_precisions, model):
+    try:
+        telemetry = tm.Telemetry()
```

Comment:
why in some places do  you provide parameters to constructor and in other not?

Reply:
because by the time we got here, the telemetry instance has already been initialized in the `main` https://github.com/openvinotoolkit/telemetry/blob/master/src/main.py#L22

Reply:
ok